spec/factories/installment.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :installment do
+    digit = Faker::Number.digit
+    value = Faker::Number.decimal(digit.to_i, 2)
+    value { value }
```

Comment:
Why?

Reply:
decimal method create an number with 'digit' length randomic to left dot and length 2 after dot 
An better aproach is Faker::Number.positive.round(2), means @RodrigoNM